### PR TITLE
Features to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move Features to class in Friendica\Content namespace. Update function calls and references.

Related to #3878

### CODE REVIEW

File: src/Content/Features.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+/**
+ * @file src/Content/Features.php
+ * @brief Features management
+ */
+namespace Friendica\Content;
+
+use Friendica\Core\Config;
+use Friendica\Core\PConfig;
+
+class Features
```
Comment (MrPetovan) [inline src/Content/Features.php]: Please rename to `Feature`.

File: src/Content/Features.php [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+						}
+					}
+				}
+				if (! $has_items) {
+					unset($arr[$k]);
+				}
+			}
+		}
+
+		call_hooks('get',$arr);
```
Comment (MrPetovan) [inline src/Content/Features.php]: This file is supposed to require include/plugin.php to call this function.
Also can you please add a space after the comma?

**Comment by MrPetovan** [comment]
After careful review, there are many other missing spaces after a comma, please fix the entire `src/Content/Feature.php` file.

**Comment by zeroadam** [comment]
That's what I get for working on this from another computer that doesn't have phpcs installed!

**Pull request merged**